app/routes/titiler/umd_glad_dist_alerts.py

```diff
@@ -52,23 +52,24 @@ async def glad_dist_alerts_raster_tile(
         description="Only show alerts until given date.",
     ),
     render_type: RenderType = Query(
-        RenderType.encoded, description="Render true color or encoded tiles"
+        RenderType.encoded,
+        description="Render true color or encoded tiles. Encoded tiles have the alert date and confidence packed in the image RGB channels for front-end interactive use, such as date filtering with supported technologies. Decoding instructions: Alert Date is calculated as `red * 255 + green`, representing days since 2020-12-31. Confidence is calculated as `floor(blue / 100)`, with values of `2` (high) or `1` (low). Intensity is calculated as `mod(blue, 100)`, with a maximum value of `55`.",
     ),
     alert_confidence: Optional[AlertConfidence] = Query(
         AlertConfidence.low,
         description="Show alerts that are at least of this confidence level",
     ),
-    tree_cover_density: Optional[int] = Query(
+    tree_cover_density_threshold: Optional[int] = Query(
         None,
         ge=0,
         le=100,
         description="Alerts in pixels with tree cover density (in percent) below this threshold won't be displayed. `umd_tree_cover_density_2010` is used for this masking.",
     ),
-    tree_cover_height: Optional[int] = Query(
+    tree_cover_height_threshold: Optional[int] = Query(
         None,
         description="Alerts in pixels with tree cover height (in meters) below this threshold won't be displayed. `umd_tree_cover_height_2020` dataset in the API is used for this masking.",
     ),
-    tree_cover_loss_cutoff: Optional[int] = Query(
+    tree_cover_loss_threshold: Optional[int] = Query(
         None,
         ge=2021,
         description="""This filter is to be used in conjunction with `tree_cover_density` and `tree_cover_height` filters to detect only alerts in forests, by masking out pixels that have had tree cover loss prior to the alert.""",
```

Comment:
**Need to change "tree_cover_density" to "tree_cover_density_threshold", and "tree_cover_height" to "tree_cover_height_threshold".